api/graphql/models/gen_models.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strconv"
 
 	"github.com/MichaelMure/git-bug/bug"
+	"github.com/MichaelMure/git-bug/entity"
```

Comment:
This file is auto generated and shouldn't be edited.
Go into _api/graphql_ directory and run `go generate` to update this file

Reply:
Resolved.